terraform/application/config/test.tfvars.json

```diff
@@ -0,0 +1,7 @@
+{
+  "cluster": "test",
+  "namespace": "srtl-test",
+  "environment": "test",
+  "deploy_azure_backing_services": false,
```

Comment:
remove deploy_azure_backing_services for test as we will use azure backing service